fr/part2.md

```diff
@@ -2,9 +2,9 @@
 
 Le protocole HTTP 1.1 est omniprésent sur Internet. De nombreux investissements ont été réalisés sur des protocoles et infrastructures tirant profit de celui-ci. À tel point que lors de l'implémentation d'un nouveau projet, il est souvent plus facile d'utiliser HTTP plutôt que de développer un nouveau protocole.
 
-## 2.1 HTTP 1.1 est énorme
+## 2.1 HTTP 1.1 est dense
 
-Lors de la création de HTTP, il fut probablement perçu comme un protocole plutôt simple et évident, ce qui, avec le temps, s'est révélé faux. HTTP 1.0 spécifié dans la RFC 1945 est une spécification de 60 pages datant de 1996. La RFC 2616 qui décrit HTTP 1.1 a été publiée trois ans plus tard en 1999 et comprend 176 pages. Puis, quand nous, à l'IETF, avons mis à jour cette spécification, elle a été répartie en six documents, avec davantage de pages au total (RFC 7230 et associées). HTTP 1.1 est énorme, compte tenu des nombreux détails, subtilités et nombreux points optionnels.
+Lors de la création de HTTP, il fut probablement perçu comme un protocole plutôt simple et évident, ce qui, avec le temps, s'est révélé faux. HTTP 1.0 spécifié dans la RFC 1945 est une spécification de 60 pages datant de 1996. La RFC 2616 qui décrit HTTP 1.1 a été publiée trois ans plus tard en 1999 et comprend 176 pages. Puis, quand nous, à l'IETF, avons mis à jour cette spécification, elle a été répartie en six documents, avec davantage de pages au total (RFC 7230 et associées). Tout compte fait, HTTP 1.1 est dense car comportant une multitude de détails, subtilités et, non dans une moindre mesure, de très nombreux points optionnels.
```

Comment:
There's a bug in this view, as the diff does not pinpoint the sentence that has been modified. Display the rich diff in order to see it properly. (It's the last sentence of this paragraph.)

Reply:
Hello @Gregordy 
"HTTP 1.1 est tres dense" (ajouter "tres").

I propose you change "car comportant" in "car comporte" ?
OR
(if you want to keep gerundiv) replace "car" for "en"

does it sound ?

cheers,
max

Reply:
Mike @machour I'd use `car il comporte`. That said, this causal relationship with `car` is missing in the english version: `By any count, HTTP 1.1 is big and includes a myriad of details`. So maybe use:
 `HTTP 1.1 est immense _et_ comporte une multitude de détails...`
?

Moreover, I don't think `by any count` is properly translated in `tout compte fait` (which in english would rather be `all things considered`). Here the idea is more about "whatever angle we look at it", so in french maybe something like `Quelle que soit la manière dont on l'observe, ...` (not a huge fan, feel free to propose something else)

Reply:
I think we can go with "Au bout du compte" or "Tout bien considéré".

To me, it denotes the acquisition of a certain knowledge about a given thing after having considered it deeply, or looked at it under all possible angles.

Yes, I will change this "car comportant" because I remember having this gut feeling when I wrote it. Feeling that told me that something was "wrong" about it, like a lack of smoothness/fluidity.